src/main/java/org/opensearch/security/ssl/transport/SSLConfig.java

```diff
@@ -36,7 +36,7 @@ public SSLConfig(final boolean sslOnly, final boolean dualModeEnabled) {
         this.sslOnly = sslOnly;
         this.dualModeEnabled = dualModeEnabled;
         if (this.dualModeEnabled && !this.sslOnly) {
-            logger.warn("opendistro_security_config.ssl_dual_mode_enabled is enabled but opendistro_security.ssl_only mode is disabled. "
+            logger.warn("opendistro_security_config.ssl_dual_mode_enabled is enabled but plugins.security.ssl_only mode is disabled. "
```

Comment:
Should refer to the actual settings that is disabled.

Reply:
I don't get this. It is referring to the actual settings that is disabled.

Reply:
Is that the setting that should be backward compatible?

Reply:
yes, `opendistro_security_config.ssl_dual_mode_enabled` is the only setting that needs to implement fallback value to be backward compatible

Reply:
So, if it is `opendistro_security.ssl_only` that is actually disabled, the warning message that refers to plugins.security.ssl_only is confusing. Can warning message refer to the actual setting that is disabled?

Reply:
This warning specified `opendistro_security_config.ssl_dual_mode_enabled` is enabled and `opendistro_security.ssl_only mode is disabled`
as you can see in line 38 `if (this.dualModeEnabled && !this.sslOnly) {`
I don't think we should change this.